### PR TITLE
Add binutils

### PR DESCRIPTION
**Pull request by CRKatri**

### CODE REVIEW

File: binutils.mk [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+			rm -rf $$pkg/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/{locale,info}; \
+		done; \
+	done
+	cp -r $(BUILD_STAGE)/binutils/x86_64-linux-gnu/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share $(BUILD_DIST)/binutils-common/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)
+	for man in $(BUILD_DIST)/binutils-common/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/man/man1/*; do \
+		mv $$man $(BUILD_DIST)/binutils-common/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/man/man1/binutils-$$(basename $$man .1 | rev | cut -d- -f1 | rev ).1; \
+	done
+	
+	# binutils.mk Sign
+	$(call SIGN,binutils-aarch64-linux-gnu,general.xml)
```
Comment (Diatrus) [inline binutils.mk:79]: Would doing a for $$target loop in the SIGN and PACK stages decrease the size of this makefile? Also, great to see you're still alive. :)

**Review by Diatrus — CHANGES_REQUESTED** [review]

**Comment by Diatrus** [comment]
Last time I tried this it didn't even get close to building on macOS. If you can, can you give it another try?

**Comment by CRKatri** [comment]
I removed the BSD targets, cause a bunch of them didn't work, I don't have a Mac or Mac VM to test building them on MacOS, sorry.

**Comment by CRKatri** [comment]
None wants binutils when you can use llvm #779

**Pull request closed without merging**